index.html

```diff
@@ -12,6 +12,14 @@
       <header>
         <a href="#" class="logo">karan</a>
         <img src="SVG/Icon - Menu.svg" alt="" class="menuIcon" />
+        <ul class="menuList">
+          <li>Portfolio</li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#Contact">Contact</a></li>
+          <li>
+            <a href="#"><img src="SVG/Mail.svg" alt="" /></a>
+          </li>
```

Comment:
- [OPTIONAL] Just a tip 💡 You can create a Mailto Link by inserting a "mailto:" after `href` 💯

> For Example:
```
<a href="mailto:your_email@example.com">
  <img src="SVG/Mail.svg" alt="" />
</a>
```

> When a user clicks on the Mailto link, the default email client on the visitor's computer opens and suggests sending a message to the email address mentioned in the Mailto link. 😉 [For more info](https://www.rapidtables.com/web/html/mailto.html)